tests/virt_autotest/virt_utils.pm

```diff
@@ -235,4 +234,63 @@ sub lpar_cmd {
     }
 }
 
+sub upload_virt_logs {
+    my ($log_dir, $compressed_log_name) = @_;
+
+    my $full_compressed_log_name = "/tmp/$compressed_log_name.tar";
+    script_run("tar cvf $full_compressed_log_name $log_dir; gzip -f $full_compressed_log_name; rm $log_dir -r", 60);
```

Comment:
Wouldn't it be shorter to simply do:
```suggestion
     my $full_compressed_log_name = "/tmp/$compressed_log_name.tar.gz"; 
    script_run("tar -czf $full_compressed_log_name $log_dir; rm $log_dir -r", 60);
```

Reply:
Yes, thanks for the suggestion. Will change to it.